### PR TITLE
Add `deps` and `recompact` tools to manual

### PR DESCRIPTION
**Pull request by adzenith**
The `deps` tool in particular is very useful to know about.

### CODE REVIEW

File: doc/manual.asciidoc [hunk 1]
```diff
@@ -279,6 +279,11 @@ http://clang.llvm.org/docs/JSONCompilationDatabase.html[JSON format] expected
 by the Clang tooling interface.
 _Available since Ninja 1.2._
 
+`deps`:: show all dependencies stored in the `.ninja_deps` file. When given a
+target, show just the target's dependencies.
+
+`recompact`:: recompact the `.ninja_deps` file.
```
Comment (nico) [inline doc/manual.asciidoc]: add `_Available since Ninja 1.4._`

File: doc/manual.asciidoc [hunk 2]
```diff
@@ -279,6 +279,11 @@ http://clang.llvm.org/docs/JSONCompilationDatabase.html[JSON format] expected
 by the Clang tooling interface.
 _Available since Ninja 1.2._
 
+`deps`:: show all dependencies stored in the `.ninja_deps` file. When given a
+target, show just the target's dependencies.
```
Comment (nico) [inline doc/manual.asciidoc]: add _Available since Ninja 1.4._

**Comment by nico** [comment]
Thanks! Can you add a note to each that they've been available since 1.4?

**Comment by adzenith** [comment]
Sure!

**Comment by nico** [comment]
Thanks!

**Pull request merged**